test/api.js

```diff
@@ -1022,3 +1022,49 @@ function generateTests(prefix, apiCreator) {
 		]);
 	});
 }
+
+function generatePassDebugTests(execArgv, expectedInspectIndex) {
+	test('pass ' + execArgv.join(' ') + ' to fork', function (t) {
+		t.plan(expectedInspectIndex === -1 ? 3 : 2);
+
+		var api = new Api({testOnlyExecArgv: execArgv});
```

Comment:
> This should stub

Not quite understand, do you mean that we should replace `process.execArgv` during test? I think, it is not acceptable to touch global state.